core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -776,7 +796,8 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
      * @memberof DotEmaComponent
      */
     reloadIframe() {
-        this.iframe.nativeElement.contentWindow?.postMessage(
+        this.cd.detectChanges();
```

Comment:
Nop! Removed!

Reply:
Are we sure that we don't need the change detection on the rest of the actions?

Reply:
If we are removing it in the rest we should keep them as they were, for the code in the iframe is ok not to have it because the iframe content doesn't depends of the change detection.